worker/task/process_event_creation.go

```diff
@@ -227,16 +237,24 @@ func findSubscriptions(ctx context.Context, endpointRepo datastore.EndpointRepos
 			subscriptions = append(subscriptions, subs...)
 		}
 	} else if project.Type == datastore.IncomingProject {
-		subs, err := subRepo.FindSubscriptionsBySourceIDs(ctx, project.UID, event.SourceID)
+		subs, err := subRepo.FindSubscriptionsBySourceID(ctx, project.UID, event.SourceID)
 		if err != nil {
-			log.Errorf("error fetching subscriptions for this source %s", err)
+			log.WithError(err).Error("error fetching subscriptions for this source")
 			return subscriptions, &EndpointError{Err: errors.New("error fetching subscriptions for this source"), delay: 10 * time.Second}
 		}
 
 		subscriptions, err = matchSubscriptionsUsingFilter(ctx, event, subRepo, subs)
 		if err != nil {
 			return subscriptions, &EndpointError{Err: errors.New("error fetching subscriptions for this source"), delay: 10 * time.Second}
 		}
+
+		cliSubs, err := subRepo.FindCLISubscriptions(ctx, project.UID)
```

Comment:
@danvixent Is there a reason we are doing this? My assumption is `FindSubscriptionsBySourceID` should return all `subs` both for endpoints and devices, and then all subs with matching filters are returned.